skills/lark-drive/references/scripts/kb_gate.py

```
@@ -0,0 +1,245 @@
+    # --- Hard gate 2: overwrite on a real draft needs explicit confirmation ---
+    if write_mode == "overwrite" and str(node.get("draft_state")) == "has_draft":
+        if node.get("overwrite_confirmed") is not True:
+            hard_reasons.append("覆盖有草稿的节点但缺少用户显式确认")
+
+    # --- Governance completeness ---
+    if not governance:
+        hard_reasons.append("缺少 6 行治理表")
+    else:
+        for field in REQUIRED_NON_EMPTY:
+            if is_empty(governance.get(field)):
+                hard_reasons.append(f"治理字段缺失：{GOVERNANCE_FIELDS[field]}")
```

Ruff (0.16.3)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)